### ISSUE TITLE
Issue with sensitive variables and Offline deployments in Powershell 2

### ISSUE THREAD

**Issue by Dalmirog** [OctopusDeploy/Issues#2065]
See http://help.octopusdeploy.com/discussions/problems/39809

Octopus version: 3.1.4

**Comment by gep13** [comment]
If you need any further help with debugging this issue, I would be happy to provide any details that you require.  Thanks!

**Comment by Jevonius** [comment]
Hi, we also see the same issue. In our case, the script works fine on Server 2012 boxes, but fails on 2008 R2 boxes running PS 2. The fix by Gary worked for us - removing the "=" from the "-sensitiveVariables" part of each Calamari line.

There is also a separate (minor) issue with the $PasswordArg line at the top - the double quote at the end is just appending an empty string onto the end, rather than a terminating speech mark. This doesn't actually cause a problem where the variable is used because it's always the last parameter and Calamari seems to handle the non-terminated string automatically.
Short version, the line:
```
$PasswordArg="-sensitiveVariablesPassword="""+ $Password + ""
```
Should be:
```
$PasswordArg="-sensitiveVariablesPassword="""+ $Password + """"
```

**Comment by gep13** [comment]
@Jevonius yes, I noticed this as well.  However, after removing the `=` sign's from each of the lines, the script worked, both with, and without the extra `"` at the end of the `$PasswordArg` variable.  You are right to bring this up though.

**Comment by gep13** [comment]
@Dalmirog any updates on this?  When can we expect this to be resolved? Thanks!

**Comment by mpobudejski** [comment]
+1

**Comment by Dalmirog** [comment]
@gep13 we've raised the priority of the issue to make sure it's in our radar

**Comment by gep13** [comment]
@Dalmirog that's great, thank you! :+1:

**Comment by zentron** [comment]
Release Note: Offline Drop script fixes for PowerShell 2.0

**Comment by gep13** [comment]
@zentron this is great news, thank you very much! :+1:

**Comment by gep13** [comment]
For anyone interested... 3.2.2 has been released.  Installing now:

https://octopus.com/downloads

**Comment by gep13** [comment]
Just tested on an offline deployment, and happy to say that it is all working as expected, no need to modify any files.  Thanks!
